core/auth/src/main/java/software/amazon/awssdk/auth/signer/AsyncAws4Signer.java

```diff
@@ -69,7 +69,7 @@ public CompletableFuture<SdkHttpFullRequest> signWithBody(SdkHttpFullRequest req
             Aws4SignerRequestParams requestParams = new Aws4SignerRequestParams(signingParams);
 
             SdkHttpFullRequest.Builder builder = doSign(request, requestParams, signingParams,
-                    new ContentChecksum(digestHex, sdkChecksum));
+                                                        new ContentChecksum(digestHex, sdkChecksum));
```

Comment:
Is the indent change here intended? Or an IDE fail?

Reply:
Not intended, but more of an IDE success. The indent is an improvement, imo.